app/presenters/publishing_api/document_collection_presenter.rb

```diff
@@ -41,20 +42,18 @@ def links
 
     def details
       {
-        first_public_at: first_public_at,
         change_history: item.change_history.as_json,
         collection_groups: collection_groups,
         body: govspeak_renderer.govspeak_edition_to_html(item),
         emphasised_organisations: item.lead_organisations.map(&:content_id),
       }.tap do |details_hash|
         details_hash.merge!(PayloadBuilder::PoliticalDetails.for(item))
+        details_hash.merge!({
+          first_public_at: PayloadBuilder::FirstPublishedAt.for(item)[:first_published_at]
```

Comment:
We're doing this multiple times. Maybe we should encapsulate this behaviour into `PayloadBuilder::FirstPublicAt`?